build_shared_vars.sh

```diff
@@ -44,21 +42,27 @@ else
     ANDROID_ROOT="$ANDROID_BUILD_TOP"
 fi
 
-YOSHINO="lilac maple poplar"
-NILE="discovery pioneer voyager"
-GANGES="kirin mermaid"
-TAMA="akari apollo akatsuki"
-KUMANO="griffin bahamut"
-SEINE="pdx201"
-
-PLATFORMS="yoshino nile ganges tama kumano seine"
+PATH=$PATH:$ANDROID_ROOT/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/bin
+PATH=$PATH:$ANDROID_ROOT/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin
 
 # Mkdtimg tool
 MKDTIMG=$ANDROID_ROOT/out/host/linux-x86/bin/mkdtimg
 
 KERNEL_TOP=$ANDROID_ROOT/kernel/sony/msm-4.14
 # $KERNEL_TMP sub dir per script
-KERNEL_TMP=${build_directory:-$ANDROID_ROOT/out/${0##*-}/kernel-tmp}
+KERNEL_TMP=$ANDROID_ROOT/out/${0##*-}/kernel-tmp
+OUT_OBJ_DEST="$KERNEL_TOP"/common-kernel
+
+arguments=khd:O:D:
+while getopts $arguments argument ; do
+    case $argument in
+        k) keep_kernel_tmp=t ;;
+        d) only_build_for=$OPTARG;;
+        O) KERNEL_TMP=$OPTARG;build_directory=$OPTARG;;
+        D) OUT_OBJ_DEST=$OPTARG;;
+        h) usage; exit 0;;
+        ?) usage; exit 1;;
+    esac
+done
 
-export PATH=$PATH:$ANDROID_ROOT/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/bin
-export PATH=$PATH:$ANDROID_ROOT/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin
+export PATH
```

Comment:
Why are you nulling the PATH?

Reply:
I'm just exporting it nulling, would be PATH=